ga4gh/datamodel/datasets.py

```diff
@@ -201,6 +246,7 @@ class FileSystemDataset(AbstractDataset):
     """
     variantsDirName = "variants"
     readsDirName = "reads"
+    featuresDirName = "sequenceAnnotations"
```

Comment:
I'll leave it as is for now, unless you guys think this is an immediate issue. Propose changing this when the wiggle (continuous) data endpoints are added.